src/map-api.js

```diff
@@ -1,22 +1,27 @@
 const request = require("request");
 
-exports.getNearPlace = (event) => {
-  request({
-    url: "https://api.foursquare.com/v2/venues/explore",
-    method: "GET",
-    qs: {
-      client_id: "CLIENT_ID",
-      client_secret: "CLIENT_SECRET",
-      ll: "40.7243,-74.0018",
-      query: "coffee",
-      v: "20170801",
-      limit: 1
-    }
-  }, function(err, res, body) {
-    if (err) {
-      console.error(err);
-    } else {
-      console.log(body);
-    }
+exports.getNearPlace = (parameter) => {
+  return new Promise((resolve, reject) => {
+    const url = "https://api.foursquare.com/v2/venues/explore",
```

Comment:
This variable can be moved out the function to the top of the file, as it doesn't change on every invocation